torchtune/utils/_version.py

```diff
@@ -23,3 +29,36 @@ def torch_version_ge(version: str) -> bool:
         True
     """
     return version in torch.__version__ or torch.__version__ >= version
+
+
+def _is_fbcode():
+    return not hasattr(torch.version, "git_version")
+
+
+def _nightly_version_ge(ao_version_str: str, date: str) -> bool:
```

Comment:
this doesn't generalize to pytorch nightly version? if's ao specific, let's include ao in the function name

Reply:
Actually it should generalize since PyTorch versions use the same format

Reply:
if so, should we change the variable name to indicate general use?